tensorflow/core/BUILD

```diff
@@ -811,7 +811,7 @@ filegroup(
 # Compiles to a trivial library on non-Android to prevent irrelevant
 # build errors. If not building this as part of an android_binary,
 # a command such as the following must be used:
-# bazel build -c opt tensorflow/core:android_tensorflow_lib \
+# bazel build --config opt tensorflow/core:android_tensorflow_lib \
```

Comment:
Wait, this is an android build -- didn't we see that here it won't work? So this should stay -c opt